.azure-pipelines/Linux-CI.yml

```diff
@@ -1,5 +1,4 @@
 trigger:
-- main
```

Comment:
Do we need at least _some_ guards on the main branch? Are the release CIs doing this?

Reply:
Yes, release CIs will still be running in the main branch.